build/dependencies.yaml

```diff
@@ -65,7 +65,7 @@ dependencies:
       match: ETCD_VERSION=
     - path: staging/src/k8s.io/sample-apiserver/artifacts/example/deployment.yaml
       match: quay.io/coreos/etcd
-    - path: test/e2e/framework/nodes_util.go
+    - path: test/e2e/util/nodes_util.go
```

Comment:
why change this?

Reply:
`go run cmd/verifydependencies/verifydependencies.go "${KUBE_ROOT}"/build/dependencies.yaml`

There is a check that is run against this file. Not fixing this will break that check.